### DISCUSSION TITLE
Metadata downloading

### DISCUSSION THREAD

**Discussion by Johnymalina** [IgnisDa/ryot#687] [Q&A] [answered]
Hi, before i raise a issue i wanted to ask here whether iam missing something. I am new user and started new instance of Ryot, all integration import and tracking is working. My only problem is Metadata update. Its not doing automatically just when i open the movie/serie than it updates all metadata. I just see list of names and generic pictures containing some form of Letter representation of a content name as a picture. I tried update all metadata and nothing. Did i missed something?

**Comment by IgnisDa** [comment]
Can you post a screenshot with what you mean.

**Reply by Johnymalina** [reply]
<img width="784" alt="image" src="https://github.com/IgnisDa/ryot/assets/74618353/71386ba3-aaf8-4593-aa3c-c48174e19d10">

**Reply by Johnymalina** [reply]
It has just some backup posters. Maybe i did explained it a bit complicated than i had to. Bui if i open the move and go back metadata downloads but only for the one movie i clicked on

**Reply by IgnisDa** [reply, marked as answer]
Yes this is done to save bandwidth. Each image is ~2MB and each movie has like 30 images. Navigating around quickly ends up bloating the database too. Data is downloaded lazily and only when needed.

**Reply by Johnymalina** [reply]
That means it works as expected and metadata download is just possible after opening the content right? Any way to force it at least for content in progress (on Dashboard page)?

**Reply by IgnisDa** [reply]
No there's no way to force it.

**Reply by Johnymalina** [reply]
Thanks for quick answer and for maintaining cool project:)

**Reply by IgnisDa** [reply]
Actually I decided to fix this. It will be done with https://github.com/IgnisDa/ryot/issues/690.
